compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -149,6 +150,16 @@ class Context(val contextId: Context.ContextId) {
   def allPackages: Map[PackageId, Ast.Package] =
     extPackages + (homePackageId -> Ast.Package(modules, extPackages.keySet, None))
 
+  private val nextTransactionSeedAndSubmissionTime = {
+    // we seed crypto.Hash.secureRandom with time to get different coid at each run.
```

Comment:
Does this not run into the entropy issues? Also why do we do this? Given that we remap them to node ids anyway, why not use a fixed seed?

Reply:
no entropy issue here. Anyway dropped